### PR TITLE
Add ddoc and controller_ha env variables

### PR DESCRIPTION
**Pull request by duynguyen**

### CODE REVIEW

File: docker-compose/docker-whisk-controller.env [hunk 1]
```diff
@@ -13,6 +13,9 @@ DB_PREFIX=local_
 DB_WHISK_ACTIONS=local_whisks
 DB_WHISK_AUTHS=local_subjects
 DB_WHISK_ACTIVATIONS=local_activations
+DB_WHISK_ACTIONS_DDOC=whisks
+DB_WHISK_ACTIVATIONS_DDOC=whisks
```
Comment (rabbah) [inline docker-compose/docker-whisk-controller.env]: You’ll want to change these to whisk.v2

Comment (duynguyen) [reply docker-compose/docker-whisk-controller.env]: oh thanks @rabbah . I was one week behind...

**Comment by rabbah** [comment]
@ddragosd another pr that we can’t merge because of a breaking change upstream. Do you think it’s worth revisiting the painful option to keep this repo functional? I defer to your judgement was just looking at the prs to see if we could fix Travis.

**Comment by csantanapr** [comment]
no longer valid, #63 made updates to make build work again

**Comment by ddragosd** [comment]
@rabbah I think it's worth generating the configurations from the `local` ansible env. This should fix the build, but on the other hand, it will generate configs that are not really used. It might make it for a good compromise until we have a release. WDYT ?

**Pull request closed without merging**